sdk/cosmos/azure-cosmos-spark_3-1_2-12/docs/configuration-reference.md

```diff
@@ -20,11 +20,10 @@ Configuration Reference:
 | `spark.cosmos.useGatewayMode`      | `false`    | Use gateway mode for the client operations  |
 | `spark.cosmos.read.forceEventualConsistency`  | `true`    | Makes the client use Eventual consistency for read operations instead of using the default account level consistency |
 | `spark.cosmos.applicationName`      | None    | Application name  |
-
+| `spark.cosmos.preferredRegionsList`      | None    | Preferred regions list to be used for a multi region Cosmos DB account. This is a comma separated value (e.g., `[eastus,westus]`) provided preferred regions will be used as hint. You should use a collocated spark cluster with your Cosmos DB account and pass the spark cluster region as preferred region.  |
```

Comment:
Would it make sense to add the link to the azure region identifiers: https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.documents.locationnames?view=azure-dotnet

Reply:
adding that now. Thanks @FabianMeiswinkel

Reply:
One interesting difference. The LocationNames use the DisplayName property of the ARM Location, like West US (with spaces), while this spark doc is showing the Name property (lowercase, no spaces). Is this expected? Do both work or just the Name for Java SDK?

Reply:
all work, we do proper name normalization. trimming, lower-casing ...

Reply:
done.